automation/deploy-bin.ts

```diff
@@ -109,8 +110,11 @@ function getPageNumbers(
 	if (!response.headers.link) {
 		return res;
 	}
-	const parse = require('parse-link-header');
+	const parse = require('parse-link-header') as typeof import('parse-link-header');
 	const parsed = parse(response.headers.link);
+	if (parsed == null) {
+		throw new Error(`Failed to parse link header: '${response.headers.link}'`);
+	}
```

Comment:
I had to add this to satisfy the typings, it just means if we do hit this branch we get a much better error message :)